extra-dev/packages/coq-quickchick/coq-quickchick.dev/opam

```diff
@@ -8,13 +8,12 @@ bug-reports: "https://github.com/QuickChick/QuickChick/issues"
 license: "MIT"
 
 build: [
-  [make "compat"]
   ["dune" "subst"] {pinned}
```

Comment:
It probably won't help with current problems, but I think this is the preferred idiom:
```suggestion
  ["dune" "subst"] {dev}
```